_posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Pulp Container 2.0.0 is Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# Pulp Ansible 0.2.0 Release Announcement
+
+The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugin.
+
+## About the Pulp Ansible content plugin
+
+If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
+
+With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
+
+* Mirror a subset of Ansible content on-premise
+* Mirror all of Ansible Galaxy’s content on-premise
+* Store private Ansible content on-premise
+* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
```

Comment:
```suggestion
* Distribute Ansible content from pulp_ansible to be installable with the ansible-galaxy CLI
```

Reply:
The suggested update is hard for me to understand. 

Would something like this work:

* Distribute Ansible content from `pulp_ansible` and perform actions with the ansible-galaxy CLI

Reply:
The title to this list states that you can do stuff with the rest api. And there you can create a distribution (that is roughly a URL to serve content).
After you did that, you can use the `ansible-galaxy` command on any consuming machine to install content from this distribution.

Reply:
This is so clear that even I understand. Thank you :D